src/GCDHelper.cs

```diff
@@ -250,14 +264,71 @@ private void ResetBar(Configuration conf) {
             SlideStart_RightTri = false;
             SlideEnd_RightTri = false;
             Slide_Background = false;
+            triggeredAlerts.Clear();
```

Comment:
I feel like the whole triggeredAlerts should be managed by the AlertManager.

Reply:
I put this in because it’s really hard to come up with a trigger that works once and only once when the bar pushes slidecast/queue lock is active.  When that’s on we either have to cache the original location of the slide/queue so we know where it was originally or come up with some way to block on subsequent iterations.  In tried using the alert itself, but that only stays in the queue for one iteration.

I’ll have a go at adding an in progress flag to the alert and leaving it in the alert manager until it is finished.  That way we can check there instead.  That wasn’t possible when I started because I was still trying to use the bar/wheel progress as the timer.  Note the notifier uses system time so we don’t care about the bar progress so it should be easy to implement. (I think).

Reply:
I left this in for now.  It almost certainly can be removed with minor adjustments, but will take significant playtime to make sure that there aren't any edge cases that arise.  Should be easy to peel out later.

Reply:
Sure, will note to remove it on the refactor!